contracts/ReClaimer.sol

```diff
@@ -0,0 +1,128 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "./AuctionSuperApp.sol";
+import "./GeoWebParcel.sol";
+import "./ERC721License.sol";
+import "./interfaces/IClaimer.sol";
+contract ReClaimer is Pausable, AccessControl, IClaimer {
+	bytes32 public constant PAUSE_ROLE = keccak256("PAUSE_ROLE");
+	bytes32 public constant RECLAIMER_ROLE = keccak256("RECLAIMER_ROLE");
+
+	/// @notice License
+	ERC721License public license;
+
+	/// @notice Auction
+	AuctionSuperApp public auctionApp;
+
+	/// @notice when the required bid amount reaches its minimum value.
+	uint256 public auctionLength;
+
+	/// @notice Emitted when a parcel is purchased
+	event ParcelReclaimed(uint256 indexed licenseId, address indexed to);
+
+	constructor() {
+		_setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+		_setupRole(PAUSE_ROLE, msg.sender);
+	}
+
+	/**
+	 * @notice Admin can update the parcel.
+	 * @param user Address of license owner to be
+	 * @param initialContributionRate Initial contribution rate of parcel
+	 * @param claimData Path of parcel to claim and Base coordinate of parcel to claim
+	 * @custom:requires RECLAIMER_ROLE
+	 */
+	function claim(
+		address user,
+		int96 initialContributionRate,
+		bytes calldata claimData
+	) external override onlyRole(RECLAIMER_ROLE) whenNotPaused returns (uint256 licenseId) {
+        if (license.ownerOf(licenseId) == address(0x0)) {
+            revert("ReClaimer: Cannot reclaim non-existent license");
+        }
+
+		uint256 licenseId  = abi.decode(claimData, (uint256));
+        uint256 startTime = auctionApp.getReclaimerStartTime(licenseId);
+        require(startTime != 0, "parcel is not for reclamation");
+		require(block.timestamp > startTime, "no relcaimer auction on this license");
+
+        // TODO implement reclaim?
```

Comment:
You're right, the super app already does the transfer. Maybe we should just move the transfer to happen here instead?